frontend/apps/reader/modules/readerbookmark.lua

```diff
@@ -55,6 +55,14 @@ function ReaderBookmark:addToMainMenu(menu_items)
             self:onShowBookmark()
         end,
     }
+    if not Device:isTouchDevice() then
```

Comment:
Should we hide this on touch devices? Pinging some touch users to see what they think:

@KenMaltby @ersidnd @cramoisi

Reply:
Merging this (we can still show it later if there's feedback towards that).